bouncer/shared/new_swap.ts

```diff
@@ -16,25 +24,54 @@ export async function newSwap(
   messageMetadata?: CcmDepositMetadata,
   brokerCommissionBps = defaultCommissionBps,
 ): Promise<void> {
+  //   const destinationAddress =
```

Comment:
?

Reply:
there are a bunch of commented out things as workaround for the SDK. They shouldn't be there when I push the newest changes.